### DISCUSSION TITLE
Init FeatureHub - Multiple API Keys

### DISCUSSION THREAD

**Discussion by fdauria** [featurehub-io/featurehub#493] [General]
### Is there a way to have multiple api_key in one implementation with node sdk?

I used the method apiKey('API_KEY') when i was creating the config, but didnt work. Later when created the context I tried to call these another flags but didnt work.

**Comment by IrinaSouth** [comment]
Hi @fdauria, are you using GET polling client or SSE to get feature updates?

**Comment by fdauria** [comment]
i'm using SSE

**Comment by rvowles** [comment]
Hi @fdauria - the SSE client is only able to support a single ApiKey per Edge client. The polling one is able to support multiple of them. It is entirely feasible to have just one repository and multiple SSE connections back to the server if you wish, would you like pseudo-code on how to do this?
